docs/launchpad/quick-start.md

```diff
@@ -329,4 +340,324 @@ Launchpad comes with a built in task to do this:
 
 ```shell
 task launchpad:pull-upstream-starter
-```
\ No newline at end of file
+```
+
+## Using Helmfile and Launchpad Charts
+
+This guide will cover two primary ways to deploy blockchain-related resources in Kubernetes using Launchpad charts: deploying all components at once using Helmfile and deploying individual components directly using Helm charts.
+
+### Prerequisites
+
+Ensure you have[helm](https://github.com/helm/helm) and [helmfile](https://github.com/helmfile/helmfile) installed on your local machine. This guide assumes familiarity with basic Helm and Helmfile operations.
+
+### Deploying using Launchpad-charts directly
+
+If you prefer to use individual components of Launchpad, such as Launchpad Charts, you can add the Launchpad Helm repository and install charts directly:
+
+```shell
+helm repo add graphops https://graphops.github.io/launchpad-charts
+helm install my-release graphops/<chart-name> --values <your-values-override.yaml>
+```
+
+#### Key Consideration
+
+Core Services: Ensure that core services like Prometheus, Grafana, Ingress, Secret Management and Storage are properly set up as they are crucial for monitoring and managing traffic to and from your applications.
```

Comment:
On the context of using helm charts directly, this "core services" list seems quite Launchpad Namespaces inspired, while I think it should be more tailored to "what do our charts actually may have an hard requirement on?".
So I think this should be replaced by mentioning those potential or mandatory requirements and their context like:
"Some of our charts assume prometheus-operator availability so... ServiceMonitors etc will fail"
"Some of our charts assume postgres-operator availability so... Postgresql CRD will fail"
"Some of our charts leverage common kubernetes core service extensions like Ingress"

afaik we have no usage of "Secret Management" in our charts, and while we do have defaults that assume openEBS, they are overridable and specifying storageClass is kind of an expect thing to do so I wouldn't mention those.